src/tracker/face_id/face_detector.py

```diff
@@ -92,7 +92,7 @@ def extract_face_from_track(
             save_tensor(cropped_image, "should_be_a_person.png")
         return self.extract_face(cropped_image)
 
-    def extract_face(self, input: torch.Tensor) -> torch.Tensor:
+    def extract_face(self, input: torch.Tensor) -> Optional[torch.Tensor]:
```

Comment:
The code has always been able to `return None` down on line 143. I suspect this means nothing is actually checking the type annotations, and we should add that in. but that's a topic for a separate PR.